src/util.h

```diff
@@ -53,6 +53,13 @@ namespace node {
 
 #define UNREACHABLE() ABORT()
 
+#define ASSIGN_OR_RETURN_UNWRAP(TYPE, PTR, OBJ, ...)                          \
+  do {                                                                        \
+    PTR = Unwrap<TYPE>(OBJ);                                                  \
```

Comment:
This change is causing a seriously strange issue where the TTY `_handle` isn't being assigned to the instance. Investigating why this is happening.